subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/userinput/DefaultUserInputHandlerTest.groovy

```diff
@@ -89,11 +88,22 @@ class DefaultUserInputHandlerTest extends Specification {
 
         and:
         input == false
+
+        where:
+        invalidInput | _
```

Comment:
💅  Consider using `invalidInput << ['', 'bla', 'y', ...]`

Reply:
I did. I tend to find this layout clearer so that I can quickly scan the cases more easily.